src/App.vue

```diff
@@ -22,7 +22,11 @@ export default {
   },
   methods: {
     getAnswer() {
-      if (this.question.indexOf('?') === -1) {
+    const index = this.question;
```

Comment:
Hi, actually I want this be a fun project and don't want to be strict :)

Reply:
ok np i got it :)